artemis-server/src/main/java/org/apache/activemq/artemis/core/server/plugin/impl/LoggingActiveMQServerPlugin.java

```diff
@@ -595,15 +595,10 @@ public void afterDeliver(ServerConsumer consumer, MessageReference reference) th
       }
    }
 
-   /**
-    * A message has been expired
-    *
-    * @param message              The expired message
-    * @param messageExpiryAddress The message expiry address if exists
-    * @throws ActiveMQException
-    */
    @Override
-   public void messageExpired(MessageReference message, SimpleString messageExpiryAddress) throws ActiveMQException {
```

Comment:
@clebertsuconic That's just a plugin implementation and I have removed a (deprecated) method implementation that it wasn't already called into the broker code, using the right API that is called instead, makes sense?